opacus/accountants/utils.py

```diff
@@ -0,0 +1,46 @@
+from opacus.accountants import get_accountant
+
+
+def get_noise_multiplier(
+    target_epsilon: float,
+    target_delta: float,
+    sample_rate: float,
+    epochs: int,
+    accountant: str = "rdp",
+    **kwargs,
+) -> float:
+    r"""
+    Computes the noise level sigma to reach a total budget of (target_epsilon, target_delta)
+    at the end of epochs, with a given sample_rate
+    Args:
+        target_epsilon: the privacy budget's epsilon
+        target_delta: the privacy budget's delta
+        sample_rate: the sampling rate (usually batch_size / n_data)
+        epochs: the number of epochs to run
+        alphas: the list of orders at which to compute RDP
+    Returns:
+        The noise level sigma to ensure privacy budget of (target_epsilon, target_delta)
+    """
+    eps = float("inf")
+    sigma_min = 0.01
+    sigma_max = 10.0
+    accountant = get_accountant(mechanism=accountant)
```

Comment:
Missed the change where you introduced this :(

First, we need to rename this to something like `create_accountant`. Otherwise it looks like there is one global accountant for the mechanism name string and you tries to retrieve it.

I understand using an optional str value in the accountant field instead of `IAccountant` instance. But why use it here? 
Can accountant of some type have hyperparameters?

Reply:
agree on `create_accountant` naming.

We don't want to pass `accountant` object from PrivacyEngine to here, because this method messes with accountant's internal state (changing accountant.steps) to do a binary search for an appropriate noise multiplier.

We don't want util method to inadvertently change the privacy state of a PrivacyEngine, so it's safer to create new object instead